assignments/B-03-81/server/app.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * this is our first file which interact with server
+ * all database connectivity is done here
+ * 
+ */
+const express = require('express');
+const bodyParser = require('body-parser');
+const passport = require('passport');
+const session = require('express-session');
+const app = express();
+const routes = require('./auth/init');
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/passport-social-auth');
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.set('port', process.env.PORT || 3000);
+app.use('/', routes);
```

Comment:
Routes are required from `./auth/init` but there are no routes present there all the routes are present in index.js . So do a proper routing and complete the flow.All passport.authenticate work will be need to be done where routes are present.